packages/sdk-js/src/feature-repository.ts

```diff
@@ -251,23 +320,56 @@ async function refreshInstance(
 async function fetchFeatures(
   instance: GrowthBook
 ): Promise<FeatureApiResponse> {
-  const [key, apiHost, clientKey] = getKey(instance);
-  const endpoint = apiHost + "/api/features/" + clientKey;
+  const { apiHost, apiRequestHeaders } = instance.getApiHosts();
+  const clientKey = instance.getClientKey();
+  const remoteEval = instance.isRemoteEval();
+  const key = getKey(instance);
+  const cacheKey = getCacheKey(instance);
 
-  let promise = activeFetches.get(key);
+  if (remoteEval) {
+    let isGbHost = false;
+    try {
+      isGbHost = !!new URL(apiHost).hostname.match(/growthbook\.io$/i);
+    } catch (e) {
+      // ignore invalid URLs
+    }
+    if (isGbHost) {
+      throw new Error("Cannot use remoteEval on GrowthBook servers");
```

Comment:
Change the message to say `GrowthBook Cloud`.  Otherwise, it may be confusing to users